src/AspNet.Security.OAuth.Introspection/Events/IntrospectTokenContext.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json.Linq;
+
+namespace AspNet.Security.OAuth.Introspection {
+    public class IntrospectTokenContext : BaseIntrospectionContext {
```

Comment:
My bad, just noticed that this is meant for an action, as if `please introspect the token` :+1: